app/models/user.rb

```diff
@@ -1,10 +1,10 @@
 class User < ApplicationRecord
   belongs_to :role
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  belongs_to :standard
+  has_many :subjects, through: :standard
+  has_many :parents, class_name: "User",foreign_key: "parent_id"
+  belongs_to :Parent, class_name: "User", optional: true
```

Comment:
@rubyboy-hub  Hi, the Parent -> has to be parent with small p

Reply:
Changed